.github/workflows/deploy-staging.yml

```diff
@@ -29,13 +29,34 @@ jobs:
     - name: npm install
       run: npm ci
 
-    - name: Build App
+    - name: Build
       run: npm run build -- --configuration staging
 
+    - uses: actions/upload-artifact@v1
+      with:
+        name: staging
+        path: ./dist
+
+  deploy:
+    name: Deploy
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Clone Repository
+      uses: actions/checkout@v2
+
+    - uses: actions/download-artifact@v1
+      with:
+        name: staging
+
+    - uses: actions/download-artifact@v1
+      with:
+        name: functions
```

Comment:
I haven't worked with github actions yet. But I would be surprised if we can download cross-repo artifacts that easily?

Reply:
Oopps... copy and paste from my testing. This is removed.